src/app/shared/libraries/accordion/accordion.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
```

Comment:
might need to prepend this id with test- in case this has to be used by the automation team